examples/angularjs/.gitignore

```diff
@@ -0,0 +1,14 @@
+node_modules/angular/
+!node_modules/angular/angular.js
+
+node_modules/angular-route/
+!node_modules/angular-route/angular-route.js
+
+node_modules/angular-mocks/
+
+node_modules/todomvc-app-css/
+!node_modules/todomvc-app-css/index.css
+
+node_modules/todomvc-common/
```

Comment:
Directories should end in `*`, otherwise they negations won't match.

https://github.com/tastejs/todomvc/commit/d674391ea8334ee663b6b050b6b88863a3cba90f#diff-6063b87db763c2a7a2731f921a3249bfR1